packages/angular-playground/package.json

```diff
@@ -23,7 +23,6 @@
   },
   "scripts": {
     "ngc": "ngc",
-    "clean": "node ./scripts/clean.js",
     "build": "npm run app:build && npm run cli:build",
     "app:build": "gulp build",
```

Comment:
Can you change this to "npm run gulp build" so that gulp does not need to be installed globally

Reply:
`"gulp build"` is the correct syntax to run the local version of gulp